src/core/app.lua

```diff
@@ -246,6 +275,16 @@ function main (options)
       breathe = latency:wrap_thunk(breathe, now)
    end
 
+   if options.ingress_drop_monitor then
+      local interval = 1e9 / 1e2   -- Every 100 milliseconds.
```

Comment:
This interval is every 10 ms, no?  1ms is 1e6 nanoseconds, so every 100ms would be 1e8 ns.